### PR TITLE
Fix HDRP scene view rendering with volumetrics and XR

### PR DESCRIPTION
**Pull request by fabien-unity**
### Purpose of this PR
Simplify volumetric slices logic to avoid dealing with double-size in XR.
Quality is slightly reduced in VR but performance will be better.

---
### Testing status

**Manual Tests**: What did you do?
- [x] Opened test project + Run graphic tests locally

**Yamato**: [running](https://yamato.prd.cds.internal.unity3d.com/jobs/78-ScriptableRenderPipeline/tree/hdrp-xr-vol-scene/.yamato%252Fupm-ci-hdrp.yml%2523All_HDRP_trunk)

### CODE REVIEW

File: com.unity.render-pipelines.high-definition/Runtime/Lighting/VolumetricLighting/VolumetricLighting.cs [hunk 1]
```diff
@@ -506,13 +499,16 @@ void PushVolumetricLightingGlobalParams(HDCamera hdCamera, CommandBuffer cmd, in
...
             cmd.SetGlobalVector(HDShaderIDs._VBufferViewportSize,               new Vector4(cvp.x, cvp.y, 1.0f / cvp.x, 1.0f / cvp.y));
-            cmd.SetGlobalInt(   HDShaderIDs._VBufferSliceCount,                 cvp.z);
-            cmd.SetGlobalFloat( HDShaderIDs._VBufferRcpSliceCount,              1.0f / cvp.z);
+            cmd.SetGlobalInt(   HDShaderIDs._VBufferSliceCount,                 sliceCount);
+            cmd.SetGlobalFloat( HDShaderIDs._VBufferRcpSliceCount,              1.0f / sliceCount);
             cmd.SetGlobalVector(HDShaderIDs._VBufferSharedUvScaleAndLimit,      currFrameParams.ComputeUvScaleAndLimit(sharedBufferSize));
             cmd.SetGlobalVector(HDShaderIDs._VBufferDistanceEncodingParams,     currFrameParams.depthEncodingParams);
             cmd.SetGlobalVector(HDShaderIDs._VBufferDistanceDecodingParams,     currFrameParams.depthDecodingParams);
-            cmd.SetGlobalFloat( HDShaderIDs._VBufferLastSliceDist,              currFrameParams.ComputeLastSliceDistance());
+            cmd.SetGlobalFloat( HDShaderIDs._VBufferLastSliceDist,              currFrameParams.ComputeLastSliceDistance(sliceCount));
```
Comment (EvgeniiG) [inline com.unity.render-pipelines.high-definition/Runtime/Lighting/VolumetricLighting/VolumetricLighting.cs]: TBH I don't really understand what's going on here. Why is it dependent on the number of eyes? It's just the distance to the last slice.

Comment (EvgeniiG) [reply com.unity.render-pipelines.high-definition/Runtime/Lighting/VolumetricLighting/VolumetricLighting.cs]: _VBufferLastSliceDist is the distance at which we start the unlit (fallback) fog. See how it's used in EvaluateAtmosphericScattering. There's no good reason it should be multiplied by 2 in VR.

Comment (fabien-unity) [reply com.unity.render-pipelines.high-definition/Runtime/Lighting/VolumetricLighting/VolumetricLighting.cs]: We took the discussion offline and figured it out :)

Comment (EvgeniiG) [reply com.unity.render-pipelines.high-definition/Runtime/Lighting/VolumetricLighting/VolumetricLighting.cs]: Okay, I understand what's going on now, my bad. It should work fine.

**Pull request merged**